src/mol-canvas3d/canvas3d.ts

```diff
@@ -416,9 +421,9 @@ namespace Canvas3D {
         let currentTime = 0;
         let drawPaused = false;
 
-        function draw(force?: boolean) {
+        function draw(force?: boolean, allowMulti?: boolean) {
```

Comment:
Since we can break some APIs with 3.0, let's change this to ``options?: { force?: boolean, allowMulti?: boolean }``?